### DISCUSSION TITLE
Croptopia

### DISCUSSION THREAD

**Discussion by BrandonPotter0** [DynamicTreesTeam/DynamicTrees#982] [Compatibility Addon Suggestions]
https://croptopia.fandom.com/wiki/Crops

https://www.curseforge.com/minecraft/mc-mods/croptopia
Over 24+ Trees?!

I saw Croptopia was supported in 1.18.2, but not in 1.201, So I would like to request compatibility for Minecraft 1.20.1 with Forge please :)

**Comment by Groupix05** [comment]
done https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-croptopia-new
